testing/kwindowsystem/APKBUILD

```diff
@@ -9,9 +9,10 @@ url="https://community.kde.org/Frameworks"
 license="MIT AND (LGPL-2.1-only OR LGPL-3.0-only"
 depends_dev="qt5-qtx11extras-dev"
 makedepends="$depends_dev extra-cmake-modules qt5-qttools-dev doxygen libxrender-dev xcb-util-keysyms-dev"
+checkdepends="xvfb-run"
 source="https://download.kde.org/stable/frameworks/${pkgver%.*}/$pkgname-$pkgver.tar.xz"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-options="!check" # Fails due to requiring running X11
+options="!check" # Test 6 hangs
```

Comment:
You can exclude the specific test from  being ran by adding `-E '(<testname>)'` to ctest.